proposals/0229-simd.md

```diff
@@ -398,38 +563,14 @@ their own set of structs, with slightly incompatible size and alignment or opera
 Even when the layout matches up, you would still need to provide conversion shims between
 each and every library you work with. There is a lot of merit in providing a single ground-truth
 for low-level vectors in the stdlib, over which libraries and programs can build additional
-operations.
-
-During the pitch phase, almost all discussion was focused on the spelling of the type names.
-There are three basic suggestions, with some passionate supporters of each:
-
-- `Float.Vector8` (this proposal). 
-
-- `Vector8<Float>` This has the virtue of leveraging the usual Swift syntax for generics, and
-arguably looking "swiftier" because of it. The downside is that although it uses the *syntax*, it
-doesn't benefit from most of the *semantics* of generics in the language, because we need to
-specialize the layout (using llvm builtin vectors) for each type/vector size to benefit from explicit
-vectorization. Because of this, it is somewhat more complex--every operation in effect ends up
-going through an extra layer of abstraction. With judicious use of `@inlinable` and 
-`@_transparent`, this should not have any performance consequences, but in my opinion it
-compromises understandability, and is a layer of complexity that we don't need.
-
-  Another argument for this convention is that it may help discoverability; if I see
-  `Vector8<Float>`, I might decide to try `Vector8<Int8>` too. On the downside, we want to
-  cap the maximum vector size at 64B; doing this with this naming scheme is somewhat
-  complicated, and involves adding a bunch of protocols for specifying element size, or else
-  you can't constrain people from doing `Vector64<Int64>`.
-
-- `Vector8.Float` This is an attempt to mirror the "desired" `Vector8<Float>` syntax
-while avoiding the extra boilerplate. The downside to this convention is that there isn't a nice
-way straight out of the box to go from element type to a vector type, so you need a parallel
-set of typenames or protocols to enable that fairly desirable operation for generic
-programming.
-
-Both the second and third option also add new members to the top-level namespace, while
-the first does not. Especially in light of the fact that we may someday be able to build 
-something like `Vector<Float, 8>`, which I find considerably more desirable, avoiding
-adding anything to that namespace for now seems to me to be the right decision.
+operations. 
+
+Dave has expressed mild concern with using the names `Vector2<T>`, etc, instead of
+`SIMDVector2<T>`. I don't think that this is a significant concern because the new `Vector2`
+implemented here should be suitably general to function as "the two-dimensional vector" of
```

Comment:
Should this be "two-element vector", or is this just terminology I'm not familiar with?